stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -538,13 +538,17 @@ public Token createPiiTokenSynchronous(@NonNull String personalId, String publis
      *  `pending`. If polling stops due to an error, the callback will be fired with the latest
      *  retrieved source and the error.
      *
+     * @deprecated Polling is being phased out in favor of webhooks. This method is not guaranteed
```

Comment:
I think it would be better to use phrasing more similar to iOS's here instead of saying "phased out". ie `"You should poll your own backend to update based on source status change webhook events it may receive"`